core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.module.ts

```diff
@@ -29,6 +29,10 @@ import { OppiaAngularRootComponent } from
 import { CkEditorCopyToolbarComponent } from 'components/ck-editor-helpers/ck-editor-copy-toolbar/ck-editor-copy-toolbar.component';
 import { TranslationLanguageSelectorComponent } from
   './translation-language-selector/translation-language-selector.component';
+import { LoginRequiredMessageComponent } from
+  './login-required-message/login-required-message.component';
+import { LoginRequiredModalContent } from
+  './modal-templates/login-required-modal.component';
```

Comment:
Optional: Import statements are exempted from max-len violations. This four lines can be two lines instead.

Reply:
Fixed.